jwala-integration-test/src/test/resources/com/cerner/jwala/ui/selenium/operations/controlJvm.feature

```diff
@@ -1,79 +1,79 @@
     And I created a jvm with the following parameters:
-        | jvmName    | CONTROL-JVM-TEST-J   |
-        | tomcat     | apache.tomcat.media |
-        | jdk        | jdk.media          |
-        | hostName   | host1                |
-        | portNumber | jvm.http.port        |
-        | group      | CONTROL-JVM-TEST-G   |
+      | jvmName    | CONTROL-JVM-TEST-J  |
+      | tomcat     | apache.tomcat.media |
+      | jdk        | jdk.media           |
+      | hostName   | host1               |
+      | portNumber | jvm.http.port       |
+      | group      | CONTROL-JVM-TEST-G  |
 
-    # create resources
+      # create resources
     And I created a JVM resource with the following parameters:
-        | group       | CONTROL-JVM-TEST-G                   |
-        | jvm         | CONTROL-JVM-TEST-J                     |
-        | deployName  | setenv.bat                      |
-        | deployPath  | jvm.setenv.resource.deploy.path |
-        | templateName| setenv.bat.tpl                  |
+      | group        | CONTROL-JVM-TEST-G              |
+      | jvm          | CONTROL-JVM-TEST-J              |
+      | deployName   | setenvFileName                  |
+      | deployPath   | jvm.setenv.resource.deploy.path |
+      | templateName | setenvTemplateFile              |
     And I created a JVM resource with the following parameters:
-        | group       | CONTROL-JVM-TEST-G                       |
-        | jvm         | CONTROL-JVM-TEST-J                         |
-        | deployName  | server.xml                          |
-        | deployPath  | jvm.server.xml.resource.deploy.path |
-        | templateName| server.xml.tpl                      |
+      | group        | CONTROL-JVM-TEST-G                  |
+      | jvm          | CONTROL-JVM-TEST-J                  |
+      | deployName   | server.xml                          |
+      | deployPath   | jvm.server.xml.resource.deploy.path |
+      | templateName | server.xml.tpl                      |
 
     And I am in the Operations tab
     And I expand the group operation's "CONTROL-JVM-TEST-G" group
 
-    # individual generate
+      # individual generate
     And I generated "CONTROL-JVM-TEST-J" JVM of "CONTROL-JVM-TEST-G" group
 
-    # test start
+      # test start
     When I click start on jvm "CONTROL-JVM-TEST-J" of the group "CONTROL-JVM-TEST-G"
     Then I see the state of "CONTROL-JVM-TEST-J" JVM of group "CONTROL-JVM-TEST-G" is "STARTED"
 
     When I click the drain button of "CONTROL-JVM-TEST-J" JVM under "CONTROL-JVM-TEST-G" group
     And I do not see an error message after clicking drain
 
-    # thread dump
+      # thread dump
     When I click thread dump of jvm "CONTROL-JVM-TEST-J" of the group "CONTROL-JVM-TEST-G"
     Then I don't see the click status tooltip
-    And I see the thread dump popup
+    And I see the thread dump popup for the jvm "CONTROL-JVM-TEST-J" with the message "thread-dump-popup-message"
```

Comment:
This is another example here - I feel like this test should be written with the expected message in the pop up and not some key to a property